[
    {
        "subtype": "channel_join",
        "user": "U015R4ZU8G5",
        "text": "<@U015R4ZU8G5> has joined the channel",
        "type": "message",
        "ts": "1592724529.287100"
    },
    {
        "subtype": "channel_join",
        "user": "U015Z90HG4U",
        "text": "<@U015Z90HG4U> has joined the channel",
        "type": "message",
        "ts": "1592796658.287400"
    },
    {
        "subtype": "channel_join",
        "user": "U015LFSA4F8",
        "text": "<@U015LFSA4F8> has joined the channel",
        "type": "message",
        "ts": "1592806487.287600"
    },
    {
        "user": "UTP7UTQHK",
        "type": "message",
        "ts": "1592807010.288700",
        "client_msg_id": "55c7eda7-b361-4154-980c-71f7e48d42c4",
        "text": "hey guys, i hit listClearedOrders every 15 mins to get my recent bets and add them to a database, but recently ive been getting a timeout error. Anyone know how to fix this\/anyone else been having the same issue?",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g03753fb84ce",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/03753fb84ce45a184d8d1622be400be3.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png",
            "first_name": "JK",
            "real_name": "JK",
            "display_name": "JK",
            "team": "T4G9NBD2M",
            "name": "jhgward94",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "QrWu",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "hey guys, i hit listClearedOrders every 15 mins to get my recent bets and add them to a database, but recently ive been getting a timeout error. Anyone know how to fix this\/anyone else been having the same issue?"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U4H19D1D2",
        "type": "message",
        "ts": "1592807757.289600",
        "client_msg_id": "cd88fd1c-47e7-459c-a83f-aa737386434f",
        "text": "by default bflw has it set to 15s, I use this endpoint a lot and have never hit a timeout, whats your connection like? Is it a requests timeout? (whats the traceback look like?)",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "137c5a3ef323",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-03-15\/6819395173841_137c5a3ef323f1944a1a_72.png",
            "first_name": "liam",
            "real_name": "liam",
            "display_name": "liam",
            "team": "T4G9NBD2M",
            "name": "liam",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "tMy",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "by default bflw has it set to 15s, I use this endpoint a lot and have never hit a timeout, whats your connection like? Is it a requests timeout? (whats the traceback look like?)"
                            }
                        ]
                    }
                ]
            }
        ]
    }
]